src/lib/autoinstall/presenters/partition.rb

```diff
@@ -0,0 +1,193 @@
+# Copyright (c) [2020] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "autoinstall/presenters/section"
+
+module Y2Autoinstallation
+  module Presenters
+    # Presenter for Y2Storage::AutoinstProfile::PartitionSection
+    #
+    # In an AutoYaST profile, a <partition> section is used to define a partition,
+    # a logical volume, a RAID member, etc.
```

Comment:
Actually this comment is slightly confusing. It would be more accurate to say something like "...to define a partition or a logical volume. In addition, for <drive> sections with disklabel=none, a <partition> section is used to define how to use the device indicated by the <drive> section."

If that is too much information, just omit the second sentence. But mixing partition and logical volumes (that are real devices) with RAID members (that are more kind of a "role") is confusing.